components/config-mgmt-service/integration_test/suggestions_test.go

```diff
@@ -734,13 +734,13 @@ func TestSuggestionsWithTableDriven(t *testing.T) {
 		// Suggestions for Policy Group
 		{"should return all policy_group suggestions",
 			request.Suggestion{Type: "policy_group"},
-			[]string{"sports", "tv_series", "RPGs", "heros", "movies", "nintendo"}},
+			[]string{"sports", "tv_series", "rpgs", "heros", "movies", "nintendo"}},
```

Comment:
Suggestions will all come back now in lowercase.

Reply:
Could this cause a problem with suggestions not being specific enough? If I have things named `a2-Dev` and `A2-dev`, the suggestion would be `a2-dev`?

Reply:
That is correct. Only the `a2-dev` suggestion would be returned. If you selected that suggestion both nodes would be shown. With this change, `A2-dev` and `a2-dev` are treated as the same term.